### PR TITLE
Update Russian translation

### PR DESCRIPTION
**Pull request by rkarabut**

### CODE REVIEW

File: config/locales/ru.yml [hunk 1]
```diff
@@ -268,14 +268,14 @@ ru:
     truncate: "&hellip;"
   push_notifications:
     favourite:
-      title: Ваш статус понравился %{name}
+      title: "Ваш статус понравился %{name}"
```
Comment (ykzts) [inline config/locales/ru.yml]: Is this double quote necessary?

File: config/locales/ru.yml [hunk 2]
```diff
@@ -268,14 +268,14 @@ ru:
...
     follow:
       title: "%{name} теперь подписан(а) на Вас"
     mention:
       action_boost: Продвинуть
       action_expand: Развернуть
       action_favourite: Нравится
-      title: Вас упомянул(а) %{name}
+      title: "Вас упомянул(а) %{name}"
```
Comment (ykzts) [inline config/locales/ru.yml]: Is this double quote necessary?

File: config/locales/ru.yml [hunk 3]
```diff
@@ -350,7 +350,7 @@ ru:
     reblogged: продвинул(а)
     sensitive_content: Чувствительный контент
   terms:
-    title: Условия обслуживания и политика конфиденциальности %{instance}
+    title: "Условия обслуживания и политика конфиденциальности %{instance}"
```
Comment (ykzts) [inline config/locales/ru.yml]: Is this double quote necessary?

**Comment by rkarabut** [comment]
Not at all, it's just how the merge conflict got resolved I think. I
usually follow the style of the original string.

24 авг. 2017 г. 17:02 пользователь "Yamagishi Kazutoshi" <
notifications@github.com> написал:

**Pull request closed without merging**